### PR TITLE
[DEVELOPER-5665] Provides a few more Metatag items/config for nodes

### PR DESCRIPTION
**Pull request by jordanpagewhite**
This provides a few more metatag settings for our node types, and edits
a few to make them consistent across node types ([node:summary] for
example).

### JIRA Issue Link
* https://issues.jboss.org/browse/DEVELOPER-5665

### Verification Process

* The og:description tag for nodes should be the actual summary (Body field summary) of the node. Previously there were node types that were using the node title as the summary (and also as og:title).
* In all instances [node:title] should be used for title metatags for consistency. Current page title should provide the same thing, so this is a bit unnecessary from an 'end result' perspective, but it is nice to see the same token being used across all node types.
* Twitter metatags should be explicitly set when possible/appropriate alongside the standard metatag set

### CODE REVIEW

File: _docker/drupal/drupal-filesystem/web/config/sync/metatag.metatag_defaults.node__article.yml [hunk 1]
```diff
@@ -8,14 +8,15 @@ id: node__article
 label: 'Content: Article'
 tags:
   referrer: no-referrer
-  og_description: '[current-page:title]'
+  og_description: '[node:summary]'
+  og_image: '[node:field_image:share], http://design.jboss.org/redhatdeveloper/website/redhatdeveloper_1_0/social/images/RHD_twitteravatar.jpg'
```
Comment (robpblake) [inline _docker/drupal/drupal-filesystem/web/config/sync/metatag.metatag_defaults.node__article.yml]: HTTPS please.

File: _docker/drupal/drupal-filesystem/web/config/sync/metatag.metatag_defaults.node__article.yml [hunk 2]
```diff
@@ -8,14 +8,15 @@ id: node__article
...
-  twitter_cards_description: '[current-page:title]'
-  twitter_cards_image: 'http://design.jboss.org/redhatdeveloper/website/redhatdeveloper_1_0/social/images/RHD_twitteravatar.jpg'
-  twitter_cards_site: '@rhdevelopers'
-  twitter_cards_title: '[current-page:title]'
+  og_site_name: '[site:name]'
+  og_title: '[node:title]'
+  twitter_cards_title: '[node:title]'
+  twitter_cards_page_url: '[node:url]/'
   twitter_cards_type: summary
+  twitter_cards_image: '[node:field_image:share], http://design.jboss.org/redhatdeveloper/website/redhatdeveloper_1_0/social/images/RHD_twitteravatar.jpg'
```
Comment (robpblake) [inline _docker/drupal/drupal-filesystem/web/config/sync/metatag.metatag_defaults.node__article.yml]: HTTPS please.

**Comment by robpblake** [comment]
retest this please

**Comment by robpblake** [comment]
rebuild this please

**Comment by jordanpagewhite** [comment]
retest this please

**Comment by jordanpagewhite** [comment]
@robpblake I addressed your feedback in the latest commit and all the tests are passing. Could you give this a review when you have a moment?

**Pull request merged**